### PR TITLE
feat:implement users section

### PR DESCRIPTION
**Pull request by LeVHoangduc**
@lenguyen-agilityio @hungnguyenhuu  Please help me to check PR and merge it.

### CODE REVIEW

File: practice-two/src/assets/css/pages/home/users.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.user__title {
+  color: var(--primary-color);
+  font-weight: var(--fw-bold);
+  line-height: 40px;
+  text-transform: uppercase;
+}
+
+.users__title--lg {
+  display: none;
+}
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/pages/home/users.css:14]: Do we have another size of user title?

Comment (LeVHoangduc) [reply practice-two/src/assets/css/pages/home/users.css:14]: At mobile
image
At desktop
image
. So, it means, for screen desktop, i will show title of desktop.

File: practice-two/src/assets/css/pages/home/users.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+.user__title {
+  color: var(--primary-color);
+  font-weight: var(--fw-bold);
+  line-height: 40px;
+  text-transform: uppercase;
+}
+
+.users__title--lg {
```
Comment (hungnguyenhuu) [inline practice-two/src/assets/css/pages/home/users.css:12]: update the naming

Comment (LeVHoangduc) [reply practice-two/src/assets/css/pages/home/users.css]: I have updated it @hungnguyenhuu

**Pull request merged**